docs/prestiti.MD

```
@@ -1,426 +1,367 @@
+    public function cancelExpiredReservations()
 }
 ```
 
-#### Cron Jobs per Automazione
+### API Endpoints
 
-```bash
-# /etc/cron.d/biblioteca
-# Controllo scadenze ogni giorno alle 8:00
-0 8 * * * php /var/www/html/biblioteca/scripts/maintenance.php
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)